syllabus/examples/training_scripts/cleanrl_procgen_plr.py

```diff
@@ -214,10 +218,14 @@ def fast_level_replay_evaluate(
     # print(eval_episode_rewards)
     mean_returns = np.mean(eval_episode_rewards)
     stddev_returns = np.std(eval_episode_rewards)
-    env_min, env_max = PROCGEN_RETURN_BOUNDS[args.env_id]
-    normalized_mean_returns = (mean_returns - env_min) / (env_max - env_min)
     policy.train()
-    return mean_returns, stddev_returns, normalized_mean_returns
+
+    if evaluator:
+        return mean_returns, stddev_returns
+    else:
+        env_min, env_max = PROCGEN_RETURN_BOUNDS[args.env_id]
+        normalized_mean_returns = (mean_returns - env_min) / (env_max - env_min)
+        return mean_returns, stddev_returns, normalized_mean_returns
```

Comment:
Same thing here, what is this code supposed to do? This function is just meant to evaluate how good the current agent is.